nltk/stem/porter.py

```diff
@@ -652,13 +652,16 @@ def stem(self, word):
         stem = word.lower()
```

Comment:
Add a default behavior to always lowercase and allow user to undo the lowercasing if deemed necessary.

```python
def stem(self, word, to_lowercase=True):
```